srcpkgs/xdg-desktop-portal/template

```diff
@@ -11,12 +11,12 @@ maintainer="Duncaen <duncaen@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="https://github.com/flatpak/xdg-desktop-portal"
 distfiles="https://github.com/flatpak/${pkgname}/releases/download/${version}/${pkgname}-${version}.tar.xz"
-checksum=883c9c9a925e48af54812b5347f546dd776ec2e27076a83d5a8126af6bafb9da
+checksum=a2fc5681b3a35078239072a98d6435a4c8404016730cab17c9febfd4ecec3610
 
 pre_configure() {
 	autoreconf -vfi
 }
 
 post_install() {
-	rm -rf "${DESTDIR}/usr/lib/systemd"
+	rm -fR "${DESTDIR}/usr/lib/systemd"
```

Comment:
Is there a reason to change that?

Reply:
Well, technically, `-R` is the official argument for recursion. `-r` is just an alias.
If you look at other commands, like `umount`, `-R` is for recursion.

Reply:
> -R
> Remove file hierarchies. See the DESCRIPTION.
> -r
> Equivalent to -R.

https://pubs.opengroup.org/onlinepubs/9699919799/utilities/rm.html

Reply:
If you're going to be pedantic, I'll pedantically note that "equivalence" and "alias" are two different concepts. There is no merit in this change.

Reply:
> -R
> Copy file hierarchies.

https://pubs.opengroup.org/onlinepubs/9699919799/utilities/cp.html

Reply:
What `cp` and `umount` do for recurrence are of no consequence. This is an `rm` invocation. The `rm` man page, from your first link, establishes that `-r` and `-R` are *equivalent*. There is no "official" argument and one is not an "alias". Equivalence means that the two alternatives can be used interchangably *in every instance*.

Stop trying to argue for this change on principle. This is a purely stylistic change with no tangible benefit that has the drawback of concealing the functional history of this line in the template. It is without merit.